en/docs/debug-the-code/index.md

```diff
@@ -0,0 +1,7 @@
+# Debug the code
+
+The sections below describe the debugging experience of the Ballerina VS Code extension, which provides multiple options and features to debug Ballerina applications.
```

Comment:
Is it correct to say sections as they are separate pages?

Reply:
changed `sections` -> `topics`.